client/src/app/core/repositories/relations.ts

```diff
@@ -126,15 +125,15 @@ export const RELATIONS: Relation[] = [
     }),
     ...makeM2O({
         OViewModel: ViewOrganisation,
-        MViewModel: ViewRole,
+        MViewModel: ViewUser,
         OField: 'roles',
-        MField: 'organisation'
+        MField: 'organisation_management_level'
```

Comment:
This is not a relation - the `user/organisation_management_level` is a string describing his/her role (e.g. the value is `can_manage_organisation`, any of https://github.com/OpenSlides/OpenSlides/blob/openslides4-dev/docs/models.yml#L114.

so if `user/organisation_management_level === 'superadmin'` the user is superadmin.

Reply:
...so just remove these relations